py-polars/polars/dataframe/_html.py

```diff
@@ -107,7 +107,7 @@ def write_header(self) -> None:
 
     def write_body(self) -> None:
         """Write the body of an HTML table."""
-        str_lengths = int(os.environ.get("POLARS_FMT_STR_LEN", "15"))
+        str_len_limit = int(os.environ.get("POLARS_FMT_STR_LEN", default=30))
```

Comment:
Maybe also use the new DEFAULT_STR_LEN_LIMIT here instead of a "magic" 30 🤓?

Reply:
We don't really have the infrastructure in place right now to share config defaults between Rust and Python, so a magic number will have to do for now.

There is a lot of refactoring that should happen in the formatting code, but I'm holding off on that for now.